backend/copr_backend/rpm_builds.py

```diff
@@ -143,8 +143,15 @@ class ArchitectureUserWorkerLimit(HashWorkerLimit):
     Copr owner (user or group).
     """
     def __init__(self, architecture, limit):
+        def _hasher(job):
+            if job.requested_arch != architecture:
+                # When _hasher(job) returns None, no limit applies, see
+                # HashWorkerLimit._groups.add() for more info.
+                return None
+            return f"{job.requested_arch}_{job.owner}"
```

Comment:
Returning `None` seems strange. Aren't we supposed to return some string, like this ^^ and something outside this class counts how many same strings are there and compare them to the limit?

Reply:
If we return None for a job, that particular one is not calculated; HashWorkerLimit uses StringCounter.add():
https://github.com/fedora-copr/copr/blob/9d0d6f333898bf2fb80fcfe090bd2525149085ea/common/copr_common/worker_manager.py#L111-L114

Reply:
I am reading the issue again, and I probably understand now. The `None` makes sense.